apps/core-app/src/main/modules/box-tool/addon/apps/app-provider.test.ts

```
@@ -198,11 +198,22 @@ vi.mock('./app-scanner', () => ({
   normalizeDisplayName: vi.fn((value: string | null | undefined) => value ?? null),
+  resolveDisplayName: vi.fn((displayName: string | null | undefined, fallbackName: string) => {
+    if (typeof displayName === 'string' && displayName && !displayName.includes('\uFFFD')) {
+      return displayName
+    }
+    return fallbackName
+  }),
   shouldUpdateDisplayName: vi.fn(
     (current: string | null | undefined, next: string | null | undefined) => {
       const normalizedCurrent = current ?? null
       const normalizedNext = next ?? null
+      if (typeof normalizedNext === 'string' && normalizedNext.includes('\uFFFD')) return false
```

ESLint

[error] Expect newline after if

(antfu/if-newline)